controller_report_xls/controllers/main.py

```diff
@@ -1,27 +1,4 @@
 # -*- coding: utf-8 -*-
-###########################################################################
```

Comment:
> Are you agree?

Yes I'm

Note: Just for the record, I don't requested replace the large-format by short-format... I just requested add a license... maybe short one (It was deleted)